cdap-ui/app/cdap/components/AppHeader/AppDrawer/AppDrawer.tsx

```diff
@@ -0,0 +1,151 @@
+    this.setState({ onNamespacePreferenceEdit: !this.state.onNamespacePreferenceEdit });
+  };
+  public render() {
+    const { classes, open, onClose, componentDidNavigate = () => null } = this.props;
+    const { isNativeLink, namespace } = this.props.context;
+    return (
+      <Drawer
+        open={open}
+        onClose={onClose}
+        className={classes.drawer}
+        disableEnforceFocus={true}
+        disableEscapeKeyDown={this.state.onNamespacePreferenceEdit}
+        ModalProps={{
+          keepMounted: true,
+        }}
+        classes={{
+          paper: classes.drawerPaper,
+        }}
+      >
+        <div className={classes.toolbar} />
+        <NamespaceDropdown
+          onNamespaceCreate={onClose}
+          onNamespacePreferenceEdit={this.toggleonNamespacePreferenceEdit}
+          tag={isNativeLink ? 'a' : Link}
+        />
+        <List component="nav" dense={true}>
+          <ListItemLink
+            component={isNativeLink ? 'a' : Link}
+            href={`/cdap/ns/${namespace}`}
+            to={`/ns/${namespace}`}
+            onClick={componentDidNavigate}
+          >
+            <ListItemText
+              disableTypography
+              classes={{ root: classes.listItemText }}
+              primary={Theme.featureNames.controlCenter}
+            />
+          </ListItemLink>
+        </List>
+        <List component="nav" dense={true}>
+          <DrawerFeatureLink
+            featureName={Theme.featureNames.pipelines}
+            featureFlagName={Theme.showPipelines}
+            featureUrl={`/ns/${namespace}/pipelines`}
+            componentDidNavigate={this.props.componentDidNavigate}
+          />
+          <DrawerFeatureLink
+            featureName={Theme.featureNames.dataPrep}
+            featureFlagName={Theme.showDataPrep}
+            featureUrl={`/ns/${namespace}/dataprep`}
+            componentDidNavigate={this.props.componentDidNavigate}
+          />
+          <DrawerFeatureLink
+            featureUrl={`/ns/${namespace}/experiments`}
+            featureFlagName={Theme.showAnalytics}
+            featureName={Theme.featureNames.analytics}
+            componentDidNavigate={this.props.componentDidNavigate}
+          />
+          <DrawerFeatureLink
+            featureUrl={`/ns/${namespace}/rulesengine`}
+            featureFlagName={Theme.showRulesEngine}
+            featureName={Theme.featureNames.rulesEngine}
+            componentDidNavigate={this.props.componentDidNavigate}
+          />
+          <DrawerFeatureLink
+            featureUrl={`/metadata/ns/${namespace}`}
+            featureFlagName={Theme.showMetadata}
+            featureName={Theme.featureNames.metadata}
+            isAngular={true}
+          />
+        </List>
+        <List component="nav" dense={true}>
+          <ListItemLink
+            component={isNativeLink ? 'a' : Link}
+            href="/cdap/administration/configuration"
+            to="administration/configuration"
```

Comment:
on react side, this project admin link have the additional `/ns` which routes to a 404 page.